src/main/java/build/buf/protovalidate/internal/celext/Format.java

```diff
@@ -157,7 +157,7 @@ private static void formatStringSafe(StringBuilder builder, Val val, boolean lis
     if (type == TypeEnum.Bool) {
       builder.append(val.booleanValue());
     } else if (type == TypeEnum.Int || type == TypeEnum.Uint) {
-      formatInteger(builder, Long.valueOf(val.intValue()).intValue());
```

Comment:
If formatInteger is no longer used anywhere, can you delete the implementation?

Reply:
Sure, will do